lib/msf/core/db_manager/http/servlet/db_export_servlet.rb

```diff
@@ -0,0 +1,35 @@
+module DbExportServlet
+
+  def self.api_path
+    '/api/v1/db-export'
+  end
+
+  def self.registered(app)
+    app.get DbExportServlet.api_path, &get_db_export
+  end
+
+  #######
+  private
+  #######
+
+  def self.get_db_export
+    lambda {
+      begin
+        opts = params.symbolize_keys
+	      opts[:path] = File.join(Msf::Config.local_directory, File.basename(opts[:path]))
```

Comment:
Would it be better to use a unique filename under the system temp location rather than `Msf::Config.local_directory`?

Reply:
I think the directory is fine, but I agree on the unique filename. I'll add that.